conf/config_template.yaml

```diff
@@ -97,8 +97,8 @@ inference:
   img_dir_or_csv_file: /path/to/csv/containing/images/list.csv
   state_dict_path: /path/to/model/weights/for/inference/checkpoint.pth.tar
   chunk_size: 512 # (int) Size (height and width) of each prediction patch. Default: 512
-  overlap: 10 # (int) Percentage of overlap between 2 chunks. Default: 10
-  heatmaps: True
+  smooth_prediction: True
+  overlap: 2 # overlap between tiles for smoothing. Must be an even number that divides chunk_size without remainder.
```

Comment:
inference overlap is in number of pixels not %

Reply:
Can we have it as a % instead? It would become uniform with [images_to_samples](https://github.com/NRCan/geo-deep-learning/blob/develop/images_to_samples.py#L214)
For images_to_samples.py, it used to be number of pixels, but we realized it was more intuitive to have it as a percentage.